src/main/java/com/tabnine/TabNineCompletionContributor.java

```diff
@@ -203,10 +203,16 @@ public void fillCompletionVariants(CompletionParameters parameters, CompletionRe
             result = result.withRelevanceSorter(CompletionSorter.defaultSorter(parameters, originalMatcher).weigh(new TabNineWeigher()));
             result.restartCompletionOnAnyPrefixChange();
             if (completions.user_message.length >= 1) {
-                String firstMsg = completions.user_message[0];
-                if (firstMsg.length() <= ADVERTISEMENT_MAX_LENGTH) {
-                    result.addLookupAdvertisement(firstMsg);
+                String details ="";
+                for (String message : completions.user_message){
+                    String currentMessage = details + message + " ";
+                    if (currentMessage.length() <= ADVERTISEMENT_MAX_LENGTH){
```

Comment:
Are we just concatenating them with a <space> and separator? That seems strange. 
Won't it seem like one long sentence?

Reply:
yes, one message could be broke into many rows and vice versa